funcx_container_service/application.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/environment")
+def env_endpoint():
+    return "TODO :)"
+
+
+if __name__ == "__main__":
```

Comment:
Go ahead and sort out mounting a config file into this app. Use the `APP_CONFIG_FILE` environment var to tell the path to load this config from (example in web service)

Reply:
I think this is done, but it's possible I misunderstood a piece -- just let me know :)